test/conventional-commits.ts

```diff
@@ -126,5 +126,55 @@ describe('ConventionalCommits', () => {
       });
       snapshot(cl.replace(/[0-9]{4}-[0-9]{2}-[0-9]{2}/g, '1665-10-10'));
     });
+
+    it('parses additional commits in footers', async () => {
+      const cc = new ConventionalCommits({
+        commits: [
+          {
+            message:
+              'chore: multiple commits\n\nfeat!: cool feature\nfix(subsystem): also a fix',
+            sha: 'abc345',
+            files: [],
+          },
+          {message: 'feat: awesome feature', sha: 'abc678', files: []},
+        ],
+        githubRepoUrl: 'https://github.com/bcoe/release-please.git',
+        bumpMinorPreMajor: true,
+      });
+      const cl = await cc.generateChangelogEntry({
+        version: 'v1.0.0',
+      });
+      snapshot(cl.replace(/[0-9]{4}-[0-9]{2}-[0-9]{2}/g, '1665-10-10'));
+    });
+
+    it('parses footer commits that contain footers', async () => {
```

Comment:
Add a test where there is a commit body with multiple paragraphs?

```
fix(foo): something

Details.

Some clarifying facts.

feat(bar): really important things
  Details.

feat(baz): some extra important things
  More details.

Reply:
done 👍